.github/workflows/publish.yml

```diff
@@ -0,0 +1,163 @@
+---
+name: Publish to pypi
+on:
+  push:
+  #On versioned releases
+    tags:
+      - v*.*.*
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      force:
+        type: choice
+        description: Retry Publish Version
+        options:
+          - No
+          - Yes
+      environment:
+        description: 'Deployment environment'
+        required: true
+        default: 'pypi'
+        type: choice
+        options:
+          - pypi
+          - testpypi
+      dryRun:
+        description: 'Dry Run deployment (set to false to deploy)'
+        required: true
+        type: boolean
+        default: true
+
+
+
+jobs:
```

Comment:
we may have an issue when we 'cut' our own releases as we don't have any way to not try and deploy when not on 'upstream'/'primary' repo and then give us a failed build.

Reply:
Can we make the tag pattern narrower so it will ignore our `-qgov.x` tags?